app/destination/page.js

```diff
@@ -1,54 +1,85 @@
+    thumbnail: "/destination/image-titan.png",
+  },
+];
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
-
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+import { useState } from "react";
 
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import { PlanetWishlistItem } from "@/components/destination/PlanetWishlistItem";
 
 export const Destinations = () => {
-  const [selectedPlanets, onAddPlanet] = useState([]);
+  const [selectedPlanets, setSelectedPlanet] = useState([]);
+  // const [isPlanetSelected, setIsPlanetSelected] =
+  //   useState(false);
+  const [numberOfPlanets, setNumberOfPlanets] = useState(0);
 
-  let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  // let isPlanetSelected = false;
+  // let numberOfPlanets = 0;
 
   const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    // Implement this function
-    // If you press the "ADD PLANET" the selected planet should display "SELECTED"
-    // And the counter should update, how many planets are selected (numberOfPlanets)
-    console.log(`You seleceted the following planet: ${name}, with the index of ${index}`);
-  }
+    setSelectedPlanet((selectedPlanets) => {
+      if (selectedPlanets.includes(name)) {
+        const updatedPlanets = selectedPlanets.filter(
+          (planet) => planet !== name
+        );
+        setNumberOfPlanets(updatedPlanets.length);
+
+        return updatedPlanets;
+      } else {
+        const updatedPlanets = [...selectedPlanets, name];
+        setNumberOfPlanets(updatedPlanets.length);
+        return updatedPlanets;
+      }
+    });
+    console.log(
+      `You seleceted the following planet: ${name}, with the index of ${index}`
+    );
+  };
 
   return (
     <div className="fullBGpicture">
       <main className="mainContent">
         <h1>Travel destinations</h1>
         <section className="card">
           <h2>Wishlist</h2>
-          {/* TASK - React 1 week 2 */}
-          {/* Display the number Of selected planets */}
-          {/* Display the "no planets" message if it is empty! */}
-          <p>No planets in wishlist :(</p>
-          <p>You have {numberOfPlanets} in your wishlist</p>
+          <p>
+            {numberOfPlanets > 0
```

Comment:
This is a very common practice all over the internet, just imagine opening your cart online and it will either:
- Display the items
- Have an icon and say your cart is empty